Calibration/LumiAlCaRecoProducers/plugins/AlcaPCCEventProducer.cc

```diff
@@ -101,6 +105,7 @@ void AlcaPCCEventProducer::fillDescriptions(edm::ConfigurationDescriptions& desc
   edm::ParameterSetDescription evtParamDesc;
   evtParamDesc.add<edm::InputTag>("pixelClusterLabel", edm::InputTag("siPixelClustersForLumi"));
   evtParamDesc.addUntracked<std::string>("trigstring", "alcaPCCEvent");
+  evtParamDesc.addUntracked<bool>("savePerROCInfo", true);
```

Comment:
Just for my own understanding: since this parameter affects heavily the physics output, should it be a `tracked` parameter ?

Reply:
Thank you, makes perfect sense. I modified the code